cfme/fixtures/vm.py

```diff
@@ -10,12 +14,49 @@
 
 
 def _create_vm(request, template, provider, vm_name):
+    doc_str = parser.get_docstrings_in_file(request.node.fspath.strpath)
```

Comment:
I would look at pytest-polarion-collect's `get_parsed_docstring`, which takes (and populates) a docstring cache.

The issue here is that we're going to parse docstrings repeatedly for every function/module scoped fixture using this method.

You can access the session through request, and create a docstrings attribute there to act as a cache of the parsed docstrings as files are read.

I would like to setup pytest-polarion-collect to generate this cache always, not just when one of the kwargs is being used (collect-only, generate-json, or meta-filter).  It's a startup cost, but then any fixtures trying to use the meta will get it for free.